sidekiq-staged_push.gemspec

```diff
@@ -30,7 +30,7 @@ Gem::Specification.new do |spec|
   spec.required_ruby_version = ">= 2.7.5"
 
   spec.add_dependency "activerecord", ">= 5.2.0"
-  spec.add_dependency "sidekiq", ">= 6.3.1"
+  spec.add_dependency "sidekiq", ">= 6.5.0", "<7"
```

Comment:
There's no need to bump the minimum version to 6.5.0 for the particular change that you made. `sidekiq/job` should already be available on 6.3.1

Reply:
Thanks for the info & explanation.
I thought about bumping the gem version, but didn't know if you'd want other stuff done/merged before prepping a release 🤷‍♂️ 

I'll revert the dependency version and play around to confirm(/refute?) this change is enough to fix #10. Either way I expect it'll be educational. 🤓 

Sounds like I can (should?) step away from the "additional changes" in this PR and let you "manage" them as you see fit—e.g., personal TODO, new/separate issue, ignore, ...

Thanks!